plugins/obs-qsv11/common_utils_linux.cpp

```diff
@@ -210,6 +210,9 @@ mfxStatus simple_gethdl(mfxHDL pthis, mfxMemId mid, mfxHDL *handle)
 
 mfxStatus simple_free(mfxHDL pthis, mfxFrameAllocResponse *response)
 {
+	if (response == NULL)
```

Comment:
Should be nullptr here probably.

Reply:
This is what i get for copying from windows lol. Done.